tests/unit_tests_old/tests_launch/test_kaniko_build.py

```diff
@@ -279,7 +279,7 @@ def test_create_kaniko_job_static(mock_kubernetes_client, runner):
         assert job["metadata"]["labels"] == {"wandb": "launch"}
         assert (
             job["spec"]["template"]["spec"]["containers"][0]["image"]
-            == "gcr.io/kaniko-project/executor:latest"
+            == "gcr.io/kaniko-project/executor:v1.8.0"
```

Comment:
why pin?

Reply:
Later versions of kaniko had api changes that cause issues with our implementation. Since they aren't being stable in implementation, I'd rather pin it then struggle to keep up to date with them